game/scripts/npc/abilities/furion_wrath_of_nature_oaa.txt

```diff
@@ -74,7 +74,7 @@
       "07"
       {
         "var_type"                                        "FIELD_INTEGER"
-        "kill_damage"                                     "4 5 6 7 8"
+        "kill_damage"                                     "6 10 14 20 30" //OAA
```

Comment:
Can you give this whole spell reasonable scaling. The numbers are all over the place.

Reply:
first of all, not buffing the damage...

kill damage is what chris, max, and I came up with. We leave it at these values for now please :)

Reply:
it doesnt matter how high kill dmg is if you cant kill a single creep...

Reply:
> it doesnt matter how high kill dmg is if you cant kill a single creep...

darkonius is working on adding to the lua. It will insta kill creeps (not ancients, dragons, or bosses) and these changes will go along with those changes

Reply:
update on ^. The numbers will be toned down obviously.